cypress/integration/features/login_management/passwordUpdate.feature

```diff
@@ -0,0 +1,48 @@
+@login @stable_test
+Feature: Password Update - Verify login functionality via password change
+
+    As a user, I want to see how app behaves when I'll change password.
+
+    Scenario: User change password, login and change again to old passwords.
+        Given I am on the dBildungscloud login page
+        Then I see Email field is visible and empty
+        Then I enter email
+        Then I see Password field is visible and empty
+        Then I enter password
```

Comment:
Is this the correct way to use "Then"? 
Isn't "Then" used for expected results of the "When"-steps?

Reply:
resolved